packages/astro/src/core/routing/manifest/create.ts

```diff
@@ -503,6 +498,13 @@ export function createRouteManifest(
 		...[...fileBasedRoutes, ...injectedRoutes, ...redirectRoutes].sort(routeComparator),
 	];
 
+	settings.buildOutput = getPrerenderDefault(config) ? 'static' : 'server';
```

Comment:
Could this be simplifed as `= config.output`?

Reply:
It could! Though I'm not yet 100% sure if this will match the actual config.

Reply:
Revisiting this now, I don't mind if you want to change this or not. I think past me was confused of the `getPrerenderDefault` meaning.